### PR TITLE
Fix/dbaas standalone

### PR DESCRIPTION
**Pull request by Heiiji**
| Question         | Answer
| ---------------- | ---
| Branch?          |  `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| License          | BSD 3-Clause

## Description

i has added some import and webpack config for fix standalone launch of dbaas-logs. I have see what has done otherwhere to fix error on launch.

### CODE REVIEW

File: packages/manager/apps/dbaas-logs/src/components/at-internet/at-internet.constant.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export default {
+  EU: {
+    config: {
+      level2: '85',
+    },
+  },
+  CA: {
+    config: {
+      level2: '	85',
```
Comment (antleblanc) [inline packages/manager/apps/dbaas-logs/src/components/at-internet/at-internet.constant.js:9]:
```suggestion
      level2: '85',
```

File: packages/manager/apps/dbaas-logs/src/components/at-internet/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import angular from 'angular';
+import ovhManagerAtInternetConfiguration from '@ovh-ux/manager-at-internet-configuration';
+
+import TRACKING from './at-internet.constant';
+
+const moduleName = 'dbaasLogsInternet';
```
Comment (antleblanc) [inline packages/manager/apps/dbaas-logs/src/components/at-internet/index.js:6]:
```suggestion
const moduleName = 'dbaasLogsAtInternet';
```

**Review by antleblanc — CHANGES_REQUESTED** [review]

File: packages/manager/apps/dbaas-logs/src/components/at-internet/at-internet.constant.js (hunk 1 above)
Comment (Heiiji) [reply packages/manager/apps/dbaas-logs/src/components/at-internet/at-internet.constant.js]: thanks, done :)

File: packages/manager/apps/dbaas-logs/src/components/at-internet/index.js (hunk 2 above)
Comment (Heiiji) [reply packages/manager/apps/dbaas-logs/src/components/at-internet/index.js]: thanks, done :)

**Review by antleblanc — APPROVED** [review]

**Pull request merged**